gradle/libs.versions.toml

```diff
@@ -75,6 +75,7 @@ playServicesLocation = "21.3.0"
 testng = "6.9.6"
 navigationTesting = "2.8.2"
 coreI18n = "1.0.0-alpha01"
+composeMaterial = "1.4.0"
 
 [libraries]
 androidx-activity-compose = { group = "androidx.activity", name = "activity-compose", version.ref = "activityCompose" }
```

Comment:
same question as for build.gradle is this needed?

Reply:
No thanks i deleted this